### ISSUE TITLE
Submit components relevant for WAI-wide tools to WAI Design Components

### ISSUE THREAD

**Issue by hidde** [w3c/wai-atag-report-tool#152]
cc @slhenry @nitedog 

In this issue, I will collect patterns that:

* do not yet exist in [WAI Design Components](https://wai-website-theme.netlify.app/components/)
* have custom CSS in the ART repository

When they exist in WAI Design Components, they no longer need custom CSS in ART, or in any other tools that use WAI Design components (like (future iterations of) WCAG EM Report Tool, FAST, Statement Generator, etc).

## Patterns to include

* [ ] Tool header
* [ ] Tool navigation
* [ ] Your Report box
* [ ] Show/hide button (for showing/hiding columns or sidebars)
* [ ] Progress bar
* [ ] Official accessibility resource link (e.g. to success criterion, Implementing pages, Understanding documents etc)
* [ ] Success Criterion evaluation form
* [ ] Evaluation results by result category
* [ ] Evaluation result table cell with colored border

**Comment by nitedog** [comment]
Good idea to collect these, and decide which ones can be moved as WAI-wide components.

**Comment by hidde** [comment]
Yes. And with that, be suitable for reuse by all of our tools, including those developed by non staff.

**Comment by slhenry** [comment]
Great, Hidde. I will schedule some time to put on my WAI website hat, and review how they will fit in other places throughout the site. Let me know when ready for that.

**Comment by hidde** [comment]
Removed from [0.5 milestone](https://github.com/w3c/wai-atag-report-tool/milestone/4), because this work is not part of a release.

**Comment by hidde** [comment]
I've added these to the WAI website theme. Next task up is to make sure we've 'using' them with the ATAG Report Tool.
